File: 04 - Angular/Galaxy.Angular/Galaxy.Angular.Client/ClientApp/src/app/shared/models/app-user-auth.ts
export class AppUserAuth {
  constructor() {
    this.IsAuthenticated = false;
    this.BearerToken = "";
  }
  UserName: string;
  BearerToken: string;
  IsAuthenticated: boolean;
  CanAccessCustomers: boolean;
  CanAddCustomer: boolean;
  CanSaveCustomer: boolean;
  CanAccessOrders: boolean;
}

File: 05 - Angular/Galaxy.Angular/Galaxy.Angular.API/Infrastructure/Security/AppUserAuth.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Galaxy.Angular.API.Infrastructure.Security
{
    public class AppUserAuth
    {
        public AppUserAuth() : base()
        {
            UserName = "Not authorized";
            BearerToken = string.Empty;
        }

        public string UserName { get; set; }
        public string BearerToken { get; set; }
        public bool IsAuthenticated { get; set; }
        public bool CanAccessCustomers { get; set; }
        public bool CanAddCustomer { get; set; }
        public bool CanSaveCustomer { get; set; }
        public bool CanAccessOrders { get; set; }
    }
}
